package.json

```diff
@@ -132,7 +132,7 @@
     "sass-loader": "7.0.0",
     "slug": "0.9.1",
     "style-loader": "0.19.1",
-    "styled-components": "3.2.6",
+    "styled-components": "^4.0.0-beta.9",
```

Comment:
Nit: We tend to prefer exact versions, without the `^`.

You can specify this when installing by doing `yarn add -DE dependency@version`. `-D` for devDependencies, `-E`- for exact version.